app/javascript/main/routes.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+// import PropTypes from 'prop-types'
+import {
+  // IndexRedirect,
+  // IndexRoute,
+  Route, 
+  Router, 
+  browserHistory
+} from 'react-router'
+import { Provider } from 'react-redux'
+// import { syncHistoryWithStore } from 'react-router-redux'
+import initializeStore from './store'
+// import { hasDirtyForm } from 'utils'
+// import { isAuthenticated } from 'lp-requests'
+import { views } from './todo'
+const TodoShow = views.TodoShow
+
+// const propTypes = {
```

Comment:
Looks like most of these comments can be cleared

Reply:
Yeah that's VS-specific stuff that probably shouldn't be used as a reference by anyone ever 😄